centreon/src/Core/Dashboard/Application/UseCase/AddDashboardToFavorites/AddDashboardToFavorites.php

```diff
@@ -85,18 +90,28 @@ public function __invoke(AddDashboardToFavoritesRequest $request): ResponseStatu
             );
 
             return new NoContentResponse();
-        } catch (\InvalidArgumentException $exception) {
+        } catch (InvalidArgumentException $exception) {
             $this->error($exception->getMessage());
+            $this->error(
+                "Error while adding dashboard as favorite for user : {$exception->getMessage()}",
+                [
+                    'user_id' => $this->user->getId(),
+                    'dashboard_id' => $request->dashboardId,
+                    'favorite_dashboards' => $favorites ?? [],
+                    'exception' => ['message' => $exception->getMessage(), 'trace' => $exception->getTraceAsString()],
+                ]
+            );
 
             return new InvalidArgumentResponse($exception);
-        } catch (\Throwable $exception) {
+        } catch (Throwable $exception) {
```

Comment:
Using the backslash means that the `Throwable` class is defined in global space, otherwise PHP will try to find the exception class in the current namespace before trying to retrieve it from global space.
```suggestion
        } catch (\Throwable $exception) {
```

Reply:
It's not a problem because there is a use ;) it's good practice to use the use at the top of files to quickly identify dependencies :) I prefer it

Reply:
Seen together, I resolve ;)